src/extensions/scratch3_translate/index.js

```diff
@@ -166,13 +163,9 @@ class Scratch3TranslateBlocks {
             return this._translateResult;
         }
 
-        // TODO: Set default to project language instead of english?
-        let urlBase = `${serverURL}translate?language=`;
-        let lang = 'en';
-        if (args.LANGUAGE) {
-            lang = args.LANGUAGE;
-        }
+        const lang = this.getLanguageCodeFromArg(args.LANGUAGE);
 
+        let urlBase = `${serverURL}translate?language=`;
         urlBase += lang;
```

Comment:
since this is now user droppable, should we also encodeUriComponent the lang too?